libtiff/tests/test_libtiff_ctypes.py

```diff
@@ -543,8 +543,8 @@ def test_copy(tmp_path):
             arr[_i, j] = 1 + _i + 10 * j
     # from scipy.stats import poisson
     # arr = poisson.rvs (arr)
-    tiff.SetField('ImageDescription', b'Hey\nyou')
-    tiff.write_image(arr, compression='lzw')
+    tiff.SetField('ImageDescription', b'Hey\nyou\x00')
```

Comment:
I suggest implementing this fix internally in the `SetField` method so that users will not need to bother adding null char at the end of image description string.

Reply:
Fixed.